easybuild/easyblocks/i/itac.py

```diff
@@ -94,7 +102,10 @@ def install_step(self):
             except OSError as err:
                 raise EasyBuildError("Directory %s can't be created: %s", tmpdir, err)
 
-            cmd = "./install.sh --tmp-dir=%s --silent=%s" % (tmpdir, silentcfg)
+            if LooseVersion(self.version) >= LooseVersion('2021'):
+                cmd = "./install.sh -a -s --eula accept --install-dir=%s" % self.installdir
```

Comment:
We should tie this acceptance to `eb --accept-eula-for=itac` instead of hardcoding accept.

Reply:
+1 on that

Reply:
+1

Reply:
Resolved!